packages/shared/components/AccountTile.svelte

```diff
@@ -27,21 +27,26 @@
     import { capitalize } from 'shared/lib/utils'
     import { wallet } from 'shared/lib/wallet'
     import { get } from 'svelte/store'
+    import { isBright } from 'shared/lib/helpers'
 
     const NEW_LINE = '\r\n'
 
     export let name = ''
     export let balance = ''
     export let balanceEquiv = ''
-    export let color = 'turquoise'
+    export let color = ''
     export let airdrop: StakingAirdrop = undefined
     export let size = 'm' // m, l
     export let hidden = false
     export let disabled = false
+    export let classes = ''
     export let onClick = (): void | string => ''
+    export let disabledHover = false
 
     $: darkModeEnabled = $appSettings.darkMode
 
+    $: textColor = isBright(color) ? 'gray-800' : 'white'
```

Comment:
`isBright(color) ? 'gray-800' : 'white'`
This piece of code is used in a lot of files so potentially could be abstracted out.